<p>Methods/Procedures should not have too many parameters.</p>
<h2>Noncompliant Code Example</h2>
<pre>
_method object.example(p1, p2, p3, p4, p5, p6, p7)
_endmethod
</pre>
<pre>
_proc@example_proc(p1, p2, p3, p4, p5, p6, p7)
_endproc
</pre>
