<p>If a local variable is declared but not used, it is dead code and should be removed. Doing so will improve maintainability as developers will
    not wonder what the variable is used for.</p>
<h2>Noncompliant Code Example</h2>
<pre>
_method exemplar.hello(name):
  _local message << "Hello " + name  # Unused variable
  print("Hello " + name)
_endmethod
</pre>
<h2>Compliant Solution</h2>
<pre>
_method exemplar.hello(name):
  _local message << "Hello " + name
  print(message)
_endmethod
</pre>
