<p>Give variable a proper descriptive name, consisting of a least 3 characters and a maximum of 32 characters (excluding prefix).</p>
<h2>Noncompliant Code Example</h2>
<pre>
    _local a << 1
    _local l_b << 1
</pre>
<h2>Compliant Solution</h2>
<pre>
    _local count << 1
    _local index << 1
</pre>